src/index.html

```diff
@@ -9,10 +9,74 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <ul class="calendar calendar--month-length-31 calendar--start-day-sun">
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
+      </ul>
```

Comment:
Instead of manually listing all the `calendar__day` elements, consider generating them dynamically using a template engine or JavaScript. This will make your code more maintainable and scalable, especially if the number of days changes.